README.md

```diff
@@ -72,11 +72,52 @@ To publish your application to Amazon S3 and Amazon CloudFront:
 
  Done!
 
+ ## Enabling federated sign-in
+
+Federated sign-in controls for Google, Facebook and Amazon are provided in the user interface by default; however, the client ids for these providers are not valid.  Dummy values are provided in the **federated** object within index.js so that the user interface controls appear.  You can remove any of the controls by deleting the appropriate keys from the **federated** object.
+
+You may also remove federated sign-in entirely by removing the 'federated={federated}' statement from the ReactDOM.render call in index.js. 
+
+Enabling federated sign-in is a three step process:
+
+  1.  Register your application with the provider(s).  
+
+      The identify providers will request information about your application, and will supply you with an application ID and other keys that your application(s) will use for authentication.  Keep in mind that some providers may supply separate application IDs for multiple applications even when these applications are sharing AWS resources. 
+
+  2.  Enable the provider for your application.
+
+      There are multiple ways of enabling a federated identity provider for your application.
+
+      *The AWS Mobile CLI* provides commands for enabling providers.
+
+      ```
+      $ awsmobile user-signin enable
+      $ awsmobile user-signin configure
```

Comment:
I would add the prompt that says Facebook sign-in or Google sign-in directly like this
```
$ awsmobile user-signin configure

? Sign-in is currently enabled, what do you want to do next Go to advance settings
? Which sign-in method you want to configure Facebook sign-in (currently disabled)
? Facebook App ID xxxxxxx
```

```
$ awsmobile user-signin configure

? Sign-in is currently enabled, what do you want to do next Go to advance settings
? Which sign-in method you want to configure Google sign-in (currently disabled)
? Google Web App Client ID aaabbbbccccccddddd
? Google Android Client ID aaaabbbcccdddddd
? Google iOS Client ID aaaaabbbbcccddddd
```

```$ awsmobile push